pipelines/resource_pipeline.go

```diff
@@ -60,14 +63,18 @@ type filters struct {
 type pipelineSpec struct {
 	ID                  string            `json:"id,omitempty" tf:"computed"`
 	Name                string            `json:"name,omitempty"`
-	Storage             string            `json:"storage,omitempty"`
+	Storage             string            `json:"storage,omitempty" tf:"force_new"`
 	Configuration       map[string]string `json:"configuration,omitempty"`
 	Clusters            []pipelineCluster `json:"clusters,omitempty" tf:"slice_set,alias:cluster"`
 	Libraries           []pipelineLibrary `json:"libraries,omitempty" tf:"slice_set,alias:library"`
-	Filters             *filters          `json:"filters"`
+	Filters             *filters          `json:"filters,omitempty"`
```

Comment:
can we not document the filters? They only apply to scala pipelines, which are not supported any more (except for a handful of legacy ones that already existed when we removed scala support).

Reply:
I can remove them completely

Reply:
@mgyucht possibly used them for BUI, don't remove without asking him first